MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
Interesting that even in their repo it varies:

https://github.com/dotnet/corefx/blob/master/src/System.Linq.Parallel/src/System/Linq/Parallel/Utils/Shared.cs#L18

https://github.com/dotnet/corefx/blob/master/src/System.Xml.XmlDocument/src/System/Xml/Dom/XmlChildEnumerator.cs#L10

So who knows what is "right" here.